spring-integration-ftp/src/main/java/org/springframework/integration/ftp/outbound/FtpMessageHandler.java

```diff
@@ -48,10 +48,15 @@ public FtpMessageHandler(FtpRemoteFileTemplate remoteFileTemplate) {
 		super(remoteFileTemplate);
 	}
 
+	@Deprecated
```

Comment:
Please, add some JavaDocs here with the `@deprecated` tag and explanation `in favor of ...`

Reply:
Added

Reply:
No reason to spam me with `Added` for all my comments.
If you agree with the change requested, you just push the commit.
We get notifications for them.
You comment only if you are not agree with us 😄